irepela/homework_7/retrying.py

```diff
@@ -0,0 +1,37 @@
+"""
+Create decorator, which will retry inner function multiple times until it passes
+"""
+import random
+
+
+def retry(retry_count=5):
+    def inner(func):
+
+        def limited_wrapper(*args, **kwargs):
+            exception = None
+            for i in range(0, retry_count):
+                try:
+                    return func(*args, **kwargs)
+                except Exception as e:
+                    exception = e
+                    print(e)
+            raise exception
```

Comment:
raise exception - in this case  python will think that error happened in 18 line but it is not true

```
raise from exception
```